src/Data/List/Properties.agda

```diff
@@ -575,6 +576,43 @@ module _ {a} {A : Set a} where
   tabulate-lookup []       = refl
   tabulate-lookup (x ∷ xs) = P.cong (_ ∷_) (tabulate-lookup xs)
 
+  tabulate-len : ∀ {n} → (f : Fin n → A) →
```

Comment:
As @gallais recently pointed out to me, this should be called `length-tabulate`.